src/rnn_api.cpp

```diff
@@ -529,7 +529,7 @@ static void LogCmdRNN(const miopenTensorDescriptor_t* xDesc,
                       const int seqLength,
                       const RNNDir_t dir)
 {
-    if(miopen::IsLoggingCmd())
+    if(miopen::IsLoggingCmd() && seqLength > 0)
```

Comment:
[Notice] line 549 `miopen::deref(xDesc[seqLength - 1])` is UB when `seqLength == 0`. Shall we MIOPEN_THROW in the callers (`miopenRNNForwardTraining()` etc)?

/cc @shurale-nkn

Reply:
This is internal function for API logging. If user passed seqLength == 0 to API this is UB at any RNN_API function because this is empty zero sized tensor.